src/components/editor/spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+import test from 'tape';
+import EditorFactory from './';
+import UpstreamEditor from 'draft-js-plugins-editor';
+import getShallowInstance from 'utils/shallow';
+
+const getState = (el) => {
+  if (el.type === 'span') {
+    return el.props.children;
+  }
+};
+
+test('Editor', t => {
+  const Editor = EditorFactory(React);
+  t.plan(2);
+
+  let instance = getShallowInstance(<Editor />);
+
+  t.equals(getState(instance.props.children[0]), 'Reading');
+  t.equals(instance.props.children[1].type, UpstreamEditor);
```

Comment:
(Correct me if I'm wrong) There does not appear to be properly rendering testing infrastructure in place, which is needed in order to test style-stripping on paste, and the click event which leads to read/write status change. So I wanted to touch base regarding it first.